bazel_tools/dev_env_package/dev_env_tool.bzl

```diff
@@ -11,15 +11,15 @@ all_files = glob(["**"])
 
 filegroup(
     name = "{tool}",
-    srcs = select({lbrace}
+    srcs = select({{
```

Comment:
Previously I didn't know how to escape it, so that a block is not treated as a key for replacement in a String.format :) It can be escaped by doubling.